src/EditorFeatures/Core/Classification/CopyPasteAndPrintingClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -150,18 +159,50 @@ public IEnumerable<ITagSpan<IClassificationTag>> GetAllTags(NormalizedSnapshotSp
                 }
             }
 
-            return SegmentedListPool.ComputeList(
+            return SegmentedListPool<ITagSpan<IClassificationTag>>.ComputeList(
                 static (args, tags) => args.cachedTags?.AddIntersectingTagSpans(args.spans, tags),
-                (cachedTags, spans),
-                _: (ITagSpan<IClassificationTag>?)null);
-        }
+                (cachedTags, spans));
 
-        private Task ProduceTagsAsync(
-            TaggerContext<IClassificationTag> context, DocumentSnapshotSpan snapshotSpan,
-            IClassificationService classificationService, ClassificationOptions options, ClassificationType type, CancellationToken cancellationToken)
-        {
-            return ClassificationUtilities.ProduceTagsAsync(
-                context, snapshotSpan, classificationService, _owner._typeMap, options, type, cancellationToken);
+            async ValueTask AddSyntacticSpansAsync(NormalizedSnapshotSpanCollection spans, SegmentedList<ITagSpan<IClassificationTag>> result, bool _)
```

Comment:
💡 `Task` is more appropriate than `ValueTask` for all of these

Reply:
done.